includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -80,6 +80,8 @@ private function getRequestParameters() {
 	private function addCachingHeaders() {
 		global $wgUser;
 
+		$this->response->setHeader( 'Vary', 'Cookie' );
```

Comment:
Actually, just checking, should this be `Accept-Encoding, Cookie`? It shouldn't remove the `Accept-Encoding` vary, and I think this would overwrite the existing header.